src/layouts/index.js

```diff
@@ -82,4 +83,4 @@ class Template extends Component {
   }
 }
 
-export default Template;
+export default ErrorBoundary(Template);
```

Comment:
Errors in the template will actually result in a totally empty/white page. This is probably not great. We should show _something_ as a fallback in the browser (in addition to logging error info to the console).

Reply:
What do you suggest as the fallback?
Maybe we can use a modal (using react-modal) here.

Reply:
We probably need to show a basic error page. It doesn't have to look super pretty. @joecritch or I can pretty it up.